### PR TITLE
ci image push

### PR DESCRIPTION
**Pull request by filariow**
- **update toolchain_build_push.sh script**
- **update GitHub Action**

### CODE REVIEW

File: .github/workflows/push.yaml [hunk 1]
```diff
@@ -91,6 +91,14 @@ jobs:
           go-version: ${{ env.GO_VERSION }}
           cache: false
 
+      - name: install operator-sdk
+        run: |
+          export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
+          export OS=$(uname | awk '{print tolower($0)}')
```
Comment (sadlerap) [inline .github/workflows/push.yaml:97]: You can pull these straight from `GOENV` and `GOARCH`.
```suggestion
          export ARCH="$(go env GOARCH)"
          export OS="$(go env GOOS)"
```

File: .github/workflows/push.yaml [hunk 2]
```diff
@@ -91,6 +91,14 @@ jobs:
...
+          export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.34.1
+          curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH}
+          chmod +x operator-sdk_${OS}_${ARCH} && sudo mv operator-sdk_${OS}_${ARCH} /usr/local/bin/operator-sdk
+
```
Comment (sadlerap) [inline .github/workflows/push.yaml:101]: I've noticed some active PRs are held up by this (see #75 for instance).  Maybe we can split this out so it can merge faster?  If you don't have the time, I can do it.

File: .github/workflows/push.yaml [hunk 3]
```diff
@@ -21,14 +21,14 @@ jobs:
       matrix:
         include:
           - directory: server
-            image_base: quay.io/konflux-workspaces/workspaces-server
+            image_base: quay.io/filario/workspaces-server
           - directory: operator
-            image_base: quay.io/konflux-workspaces/workspaces-operator
```
Comment (sadlerap) [inline .github/workflows/push.yaml:26]: You'll need to revert this change

File: .github/workflows/push.yaml [hunk 4]
```diff
@@ -3,7 +3,7 @@ name: Build container images
 on: # yamllint disable-line rule:truthy
   push:
     branches: [main]
-  pull_request_target:
```
Comment (sadlerap) [inline .github/workflows/push.yaml:6]: This also needs to be reverted; we need to have the `pull_request_target` trigger for things to work.

File: hack/workspaces_install.sh [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-export QUAY_NAMESPACE=${QUAY_NAMESPACE:-workspaces}
+export QUAY_NAMESPACE=${QUAY_NAMESPACE:-filario}
```
Comment (sadlerap) [inline hack/workspaces_install.sh:5]: Maybe fix this while we're at it?
```suggestion
export QUAY_NAMESPACE=${QUAY_NAMESPACE:-konflux-workspaces}
```

**Review by sadlerap — COMMENTED** [review]
A few small things, otherwise lgtm

File: .github/workflows/push.yaml (hunk 3 above)
Comment (filariow) [reply .github/workflows/push.yaml]: done

File: .github/workflows/push.yaml (hunk 4 above)
Comment (filariow) [reply .github/workflows/push.yaml]: done

File: hack/workspaces_install.sh (hunk 5 above)
Comment (filariow) [reply hack/workspaces_install.sh]: done

**Review by sadlerap — APPROVED** [review]
lgtm, let me know when you think this is okay to merge
